src/main/java/TaskList.java

```diff
@@ -0,0 +1,63 @@
+public class TaskList {
+    private Task[] taskList;
+    private int taskCount;
+
+    public TaskList() {
+        taskList = new Task[100];
+        taskCount = 0;
+    }
+
+    // Adds task to taskList using the Command object
+    public String addTask(Command command) {
+        // First argument will always be the description of the task
+        String taskName = command.getArgument(0).trim();
+        // Extra argument for Deadline and Event
+        String extraArg;
+
+        switch(command.getCommand().toLowerCase()) {
```

Comment:
Good implementation of the switch statement to handle individual states of operation!